### PR TITLE
ci(gcb): Publish Py3 images with '-py3' suffix

### PR DESCRIPTION
**Pull request by BYK**

### CODE REVIEW

**Review by mattrobenolt — APPROVED** [review]

File: docker/cloudbuild.yaml [hunk 1]
```diff
@@ -107,6 +106,27 @@ steps:
...
+  args:
+  - '-e'
+  - '-c'
+  - |
+    # Only push to Docker Hub from master
+    [ "$BRANCH_NAME" != "master" ] && exit 0
+    # Need to pull the image first due to Kaniko
+    docker pull us.gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA-py3
+    echo "$$DOCKER_PASSWORD" | docker login --username=sentrybuilder --password-stdin
+    docker tag us.gcr.io/$PROJECT_ID/sentry:$COMMIT_SHA getsentry/sentry:$SHORT_SHA-py3
```
Comment (mattrobenolt) [inline docker/cloudbuild.yaml:124]: Doesn't $COMMIT_SHA need a py3 suffix here?

Comment (mattrobenolt) [reply docker/cloudbuild.yaml:124]: Nevermind, I see why.

Comment (BYK) [reply docker/cloudbuild.yaml:124]: it does 🤦🏻

Comment (mattrobenolt) [reply docker/cloudbuild.yaml]: Then I clearly am not understanding how any of this works, haha

Comment (BYK) [reply docker/cloudbuild.yaml]: It's essentially a `mv` command. `docker tag <current_on_gcb> <what_we_want_on_docker_hub>`. Both need to have `-py3` as that's how we build the Py3 variant in earlier steps.

**Pull request merged**